cmd/system-probe/modules/security_runtime.go

```diff
@@ -0,0 +1,22 @@
+package modules
```

Comment:
missing code header

Reply:
Are you talking about the license header or about Go build constraints

Reply:
Done